dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -526,19 +526,30 @@
    <!-- datacite:rights -->
    <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_accessrights.html -->
     <xsl:template match="doc:element[@name='dc']/doc:element[@name='rights']/doc:element" mode="datacite">
+        <xsl:variable name="rightsValue" select="doc:field[@name='value']/text()"/>
```

Comment:
There is still an issue here since the select will only result in the first dc.rights value for a particular language being tested later in resolveRightsURI. In testing I added dc.rights[en_US]="Some test value" followed by dc.rights[en_US]="open access" and the datacite:rights element in the output was missing. Reversing the value order produced the correct result.